test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/MaxRequestBufferSizeTests.cs

```diff
@@ -245,11 +245,6 @@ public static IEnumerable<object[]> LargeUploadData
         private static Socket CreateSocket(int port)
         {
             var socket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
-
-            // Timeouts large enough to prevent false positives, but small enough to fail quickly.
-            socket.SendTimeout = 10 * 1000;
```

Comment:
We thought this was causing problems but it wasn't really the case - we're seeing the same failures on dev. It's most likely slowness caused by having the pipelines code embedded into Kestrel and compiled for debug. This change makes those failures more pronounced because it moves a lot of tests into the functional tests project, so there's more stuff running at the same time and that may be slowing things down even more and causing timeouts.

@davidfowl and I discussed yesterday about re-writing this test in a different way. We already have several tests that watch the logs for certain events, and we could do the same thing here and wait until we see "connection paused" and "connection resumed" logs, instead of being dependent on time as this test is right now.